buildconf

```diff
@@ -351,7 +351,7 @@ if test -f m4/libtool.m4; then
 fi
 
 echo "buildconf: running aclocal"
-${ACLOCAL:-aclocal} -I m4 $ACLOCAL_FLAGS || die "aclocal command failed"
+${ACLOCAL:-aclocal} -I m4 || die "aclocal command failed"
```

Comment:
Is it possible there will still be an ACLOCAL_FLAGS? Just because we are no longer adding to it doesn't mean it can't still be used right?

Reply:
while it could be possible someone is setting it in their environment before calling ./buildconf, it is not used that way in the CI and that use will be still logged by tests/testcurl.pl (which might be something to cleanup once this is released more broadly)

as it was historically used to set the include path, which is already set in buildconf directly, and this should only affect people building their own configure, I find that unlikely though.

Reply:
I think the `ACLOCAL_FLAGS` should be left alone. It doesn't hurt anyone and it can be used by someone. Now or later.

Reply:
anyone that needs a custom aclocal invocation could instead set ACLOCAL

if the intent is to allow passing custom flags, it would make more sense to use instead IMHO:

  ```${ACLOCAL_FLAGS:--I m4}```

and maybe even use a custom namespace to avoid accidental use like BUILCONF_ACLOCAL_FLAGS.

Reply:
Sure, but if someone is already using this feature, your edit here breaks their script without a very good justification...